src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -731,11 +739,24 @@ export default class GeFormRenderer extends LightningElement{
                 this.handleElevateAPIErrors(errors);
                 return;
             }
+        }  else if (this.isGiftAuthorized()) {
+            try {
+                await validateAuthorizedGiftEdit({'dataImport': dataImportFromFormState});
```

Comment:
Didn't know we could make imperative apex calls with params in `''`. Interesting!

Reply:
Thanks for drawing attention here. I went ahead and removed the `''` to align with how we're passing parameters into other apex methods within BGE.